examples/workers/script-upload.ts

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env -S npm run tsn -T
+
+import Cloudflare from 'cloudflare';
+
+// Requires environment variables CLOUDFLARE_API_KEY & CLOUDFLARE_EMAIL
+const client = new Cloudflare();
+// replace with your account id
+const accountID = '023e105f4ecef8ad9ca31a8372d0c353';
```

Comment:
let's use `process.env["CLOUDFLARE_ACCOUNT_ID"]` to prevent having hardcoded IDs in the examples. 

```suggestion
const accountID = process.env["CLOUDFLARE_ACCOUNT_ID"];
```

Reply:
Updated!